Index: src/java.desktop/share/classes/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/java.desktop/share/classes/module-info.java	(revision d29d53cac11ac94a72d9087e315ae99b31572cb8)
+++ src/java.desktop/share/classes/module-info.java	(date 1599879923220)
@@ -109,10 +109,7 @@
     // see make/GensrcModuleInfo.gmk
     exports sun.awt to
         jdk.accessibility,
-        jdk.unsupported.desktop,
-        jcef;
-
-    exports java.awt.peer to jcef;
+        jdk.unsupported.desktop;

     exports java.awt.dnd.peer to jdk.unsupported.desktop;
     exports sun.awt.dnd to jdk.unsupported.desktop;
Index: modules.list
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- modules.list	(revision d29d53cac11ac94a72d9087e315ae99b31572cb8)
+++ modules.list	(date 1599879940315)
@@ -62,6 +62,5 @@
 jdk.xml.dom,
 jdk.zipfs,
 jdk.hotspot.agent,
-jcef,
 jetbrains.api,
 jetbrains.api.impl
